packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-in/nodejs/integrationsteps.md

```diff
@@ -1,2 +1,111 @@
+## Integration steps
 
-Node.js sample code for this use case is coming soon. See [Okta Auth JS SDK](https://github.com/okta/okta-auth-js/blob/master/README.md) readme for further information.
+### Step 1: Build a sign-in page on the client
+
+Build a sign-in page that captures both the username and password.
+
+For example:
+
+<div class="common-image-format">
+
+![Sign in screenshot](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-on-screenshot-sign-in.png
+ "Sign in screenshot")
+
+ </div>
+
+### Step 2: Authenticate user credentials
+
+When the user initiates the sign-in process, your app needs to create a new `OktaAuth` object, which in the SDK sample app's `login.js` file is `authClient`, and set its `username` and `password` properties to the values entered by the user. Send this object to the `idx.authenticate` function to authenticate the user. See [idx.Authenticate](https://github.com/okta/okta-auth-js/blob/master/docs/idx.md#idxauthenticate) for more information.
+
+```JavaScript
+router.post('/login', async (req, res, next) => {
+  const { username, password } = req.body;
+  const authClient = getAuthClient(req);
+  const transaction = await authClient.idx.authenticate({
+    username,
+    password,
+  });
+  handleTransaction({ req, res, next, authClient, transaction });
+});
+```
+
+### Step 3: Handle the response from the sign in
+
+The application handles the response from the authentication call using the `handleTransaction` function as shown in the SDK sample application `handleTransaction.js` file. The `transaction` parameter is the `IdxStatus` passed in through the response from Okta.
```

Comment:
```suggestion
The application handles the response from the authentication call by using the `handleTransaction` function as shown in the SDK sample application `handleTransaction.js` file. The `transaction` parameter is the `IdxStatus` that is passed in through the response from Okta.
```

Reply:
Extending your suggestion: "...is the `IdxStatus` value that is passed in ..."